frontend/src/lib/logic/inAppPrompt/inAppPromptLogic.tsx

```diff
@@ -399,7 +399,11 @@ export const inAppPromptLogic = kea<inAppPromptLogicType>([
             const valid = []
             for (const sequence of values.sequences) {
                 // for now the only valid rule is related to the pathname, can be extended
-                const isMatchingPath = sequence.rule.path.must_match.some((value) => wcmatch(value)(pathname))
+                const { must_match } = sequence.rule.path
+                if (must_match.includes('/*')) {
+                    must_match.push('/**')
```

Comment:
For reviewer, this fixes a bug where something like `/helloworld` would pass but `/hello/world` would not match a wildcard path